input/resources/StructureDefinition-subjectlist-measurereport-deqm.xml

```diff
@@ -0,0 +1,428 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<StructureDefinition xmlns="http://hl7.org/fhir">

```

Comment:
```suggestion
<StructureDefinition xmlns="http://hl7.org/fhir">
	<meta>
		<profile value="http://hl7.org/fhir/us/davinci-deqm/StructureDefinition/subjectlist-measurereport-deqm" />
	</meta>
```

Reply:
the meta.profile value is not needed since this is a creation of a profile and not an example

Reply:
the meta.profile value is not needed since this is a creation of a profile and not an example